components/hal/adc_hal.c

```diff
@@ -10,6 +10,7 @@
 #include "hal/assert.h"
 #include "soc/lldesc.h"
 #include "soc/soc_caps.h"
+#include "esp_log.h"
```

Comment:
we don't allow to use "esp_log.h" in the hal component, instead, we suggest using "hal/log.h" in the hal.